pkg/lookup/lookup.go

```diff
@@ -82,6 +83,29 @@ func GetTemplates(
 	return templates
 }
 
+// GetMandatoryTemplates returns all ModuleTemplates for modules which are marked as mandatory with the given channel.
+func GetMandatoryTemplates(ctx context.Context, kymaClient client.Reader) (ModuleTemplatesByModuleName,
+	error,
+) {
+	moduleTemplateList := &v1beta2.ModuleTemplateList{}
+	if err := kymaClient.List(ctx, moduleTemplateList); err != nil {
```

Comment:
I suggest we create an issue to introduce a label for this mandatory moduletemplate, We can implement some logic in mutationwebhook, when a mandatory module created in cluster, add a label for it, then we can change this list based on label, so that we can also remove that customized filtering in the following steps.

Reply:
https://github.com/kyma-project/lifecycle-manager/issues/1198

Can be covered with this issue